src/main/java/teammates/storage/api/FeedbackResponsesDb.java

```diff
@@ -399,6 +399,15 @@ private FeedbackResponse getFeedbackResponseEntity(String feedbackResponseId) {
                 .list();
     }
 
+    /**
+     * Get number of submitted responses for a question.
+     */
+    public int getSubmittedResponseCountForQuestion(String questionId) {
+        return load()
+                .filter("feedbackQuestionId =", questionId)
```

Comment:
Can check if this filter reads into the document? And what cost will this bring?
Can use the mock script to populate some data and test the cost of this operation.